### ISSUE TITLE
A wrong path to a non-existence src file causes ICE

### ISSUE THREAD

**Issue by Y-Nak** [ethereum/fe#641]
### What is wrong?

Please include information like:

* full output of the error you received
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { depth: 0, inner: Io { path: Some("wrong_path"), err: Os { code: 2, kind: NotFound, message: "No such file or directory" } } }', crates/fe/src/main.rs:160:27
stack backtrace:
   0: rust_begin_unwind
             at /rustc/8cdb3cd94efece1e17cbd8f6edb1dc1a482779a0/library/std/src/panicking.rs:577:5
   1: core::panicking::panic_fmt
             at /rustc/8cdb3cd94efece1e17cbd8f6edb1dc1a482779a0/library/core/src/panicking.rs:135:14
   2: core::result::unwrap_failed
             at /rustc/8cdb3cd94efece1e17cbd8f6edb1dc1a482779a0/library/core/src/result.rs:1737:5
   3: core::result::Result<T,E>::unwrap
   4: fe::build_ingot_filestore_for_dir
   5: fe::main
   6: core::ops::function::FnOnce::call_once
```

* what command you ran
`RUST_BACKTRACE=1 cargo run --features solc-backend wrong_path`

**Comment by Narasimha1997** [comment]
I can work on fixing this.

**Comment by Narasimha1997** [comment]
#645

**Comment by Y-Nak** [comment]
This was fixed in #645, thanks @Narasimha1997!
